src/material-experimental/mdc-paginator/paginator.scss

```diff
@@ -88,6 +88,10 @@ $button-icon-size: 28px;
   .mat-mdc-icon-button[disabled] .mat-mdc-paginator-icon,
   .mat-mdc-paginator-icon {
     fill: currentColor;
+
+    // On Chromium browsers the `currentColor` blends in with the
+    // background for SVGs so we have to fall back to `CanvasText`.
+    fill: CanvasText;
```

Comment:
Should there be a conditional here?

Reply:
There shouldn't, because some browsers don't support `CanvasText`.